### ISSUE TITLE
Number input with wrong value in floating label is overlapped by label

### ISSUE THREAD

**Issue by alexeypankov** [ionic-team/ionic#7037]
#### Short description of the problem:

Number input with wrong value in floating label is overlapped by label

**Steps to reproduce:**
1. Add ion-input with type 'number'
2. Enter not valid value '123-123' or 'eee'

```
<ion-item>
  <ion-label floating>Username</ion-label>
  <ion-input type="number"></ion-input>
</ion-item>
```

**Other information:**
Checked on Firefox and Chrome browsers and on Android 5.1.1.

https://cdn.discourse.org/ionicframework/uploads/default/original/3X/7/1/71e771ef927788582a9f234f82ad1ffbc86151db.png

**Which Ionic Version?** 
2.0.0-beta.9
#### Plunker that shows an example of your issue

**For Ionic 2 issues** - http://plnkr.co/edit/QalAAIN5cBl7cY2LdHUv?p=preview

**Run `ionic info` from terminal/cmd prompt:** (paste output below)
Cordova CLI: 6.2.0
Ionic CLI Version: 2.0.0-beta.30
Ionic App Lib Version: 2.0.0-beta.16
OS: Distributor ID: Ubuntu Description: Ubuntu 15.10 
Node Version: v0.12.7

**Comment by kleeb** [comment]
any news?
still occurs on 3.3.0

**Comment by theHellyar** [comment]
Yes still issue in 3.4.2 can we get this fixed?

**Comment by theHellyar** [comment]
not fixed in 3.5.3...

my current solution is to use the pattern="[0-9]*" to force the number keypad to show..not ideal...
